.tekton/kueue-operator-1-0-pull-request.yaml

```diff
@@ -29,6 +29,7 @@ spec:
   - name: build-platforms
```

Comment:
We are not going to release this as part of release-1.0.

Can you revert the changes for tekton in the 1-0 pull and push yamls?